provisioning/provision.sh

```diff
@@ -53,6 +53,7 @@ virtualenv /dissemin/.vm_venv --no-site-packages -p $(which python2.7)
 # We install dependencies in the virtualenv
 req_files=(requirements.txt requirements-dev.txt)
 
+/dissemin/.wm_venv/bin/pip install --upgrade setuptools
```

Comment:
Sorry for the mistake.
...But now I don't understand why it worked on my computer ^^'